addons/findcasts.py

```diff
@@ -10,12 +10,11 @@
     if arg.startswith('-'):
         continue
 
-    print('Checking ' + arg + '...')
-    data = cppcheckdata.parsedump(arg)
+    print("Checking %s..." % arg)
```

Comment:
```suggestion
    print('Checking %s...' % arg)
```
To stay consistent I would not use `"` instead of `'`.
I am also not sure why you changed this line. The output is the same and such small strings concatenated together with `+` are no problem w.r.t. performance or memory consumption normally. But they are often better readable.

Reply:
OK. Thanks.